e2e-tests/tfm/cypress/e2e/journeys/case-underwriting/assign-lead-underwriter.spec.js

```diff
@@ -19,15 +19,15 @@ context('Case Underwriting - Assign lead underwriter (MIA only)', () => {
   const underwriterFullName = `${UNDERWRITER_1.firstName} ${UNDERWRITER_1.lastName}`;
 
   before(() => {
-    cy.getUser(UNDERWRITER_MANAGER_1.username, T1_USER_1).then((userObj) => {
+    cy.getUser(UNDERWRITER_MANAGER_1.username).then((userObj) => {
       underwriterManager1UserId = userObj._id;
     });
 
-    cy.getUser(UNDERWRITER_MANAGER_2.username, T1_USER_1).then((userObj) => {
+    cy.getUser(UNDERWRITER_MANAGER_2.username).then((userObj) => {
       underwriterManager2UserId = userObj._id;
     });
 
-    cy.getUser(UNDERWRITER_1.username, T1_USER_1).then((userObj) => {
```

Comment:
RE these `getUser` changes - unless i'm missing something, It seems that `getUser` only accepts 1 username. Any idea why 2x usernames were being passed here?

Reply:
Old command was T1_USER_1 to login to TFM-API and then call TFM-API get /users/:username. Looks like endpoint GET by username was used just for testing.

Current getUser commands gets user directly from database, no login is required.